contracts/TokenListManager.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {ITokenListManager} from "./ITokenListManager.sol";
+
+contract TokenListManager is ITokenListManager {
+
+    // Storage the token address to whitelist or blacklist
+    address[] private whitelistedTokensArray;
+    address[] private blacklistedTokensArray;
+
+    // Use mapping to check if token in whitelist
+    mapping(address => bool) private whitelistedTokens;
+    // Use mapping to check if token in blacklist
+    mapping(address => bool) private blacklistedTokens;
+
+    // Implement the addTokens method and add tokens to whitelist
+    function addTokens(address[] calldata tokens) external override {
```

Comment:
**Must** check privilege before adding a token. Otherwise, some malicious token may be added.